packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -681,38 +771,40 @@ class HeapTreeViewState extends State<HeapTree>
     clearSearchField(controller);
   }
 
-  bool get _isSearchable =>
-      controller.snapshots.isNotEmpty ||
-      controller.isAllocationMonitorLeafSelected;
+  bool get _isSearchable {
+    // Snapshot exist or 'Allocations' tab is selected.
+    return controller.snapshots.isNotEmpty || true;
+    //tabController?.index == allocationsTabIndex;
+  }
 
-  Widget _buildSearchFilterControls() {
-    return Row(
-      mainAxisSize: MainAxisSize.min,
-      mainAxisAlignment: MainAxisAlignment.end,
-      children: [
-        Container(
-          // TODO(terry): Use a more adaptive layout than forcing to 300.0
-          width: defaultSearchTextWidth,
-          height: defaultTextFieldHeight,
-          child: buildAutoCompleteSearchField(
-            controller: controller,
-            searchFieldKey: memorySearchFieldKey,
-            searchFieldEnabled: _isSearchable,
-            shouldRequestFocus: _isSearchable,
-            onSelection: selectTheMatch,
-            onHighlightDropdown: highlightDropdown,
-          ),
-        ),
-        const SizedBox(width: denseSpacing),
-        FilterButton(
-          key: filterButtonKey,
-          onPressed: _filter,
-          // TODO(kenz): implement isFilterActive
-          isFilterActive: false,
+  Widget _buildSearchWidget(GlobalKey<State<StatefulWidget>> key) => Container(
+        // TODO(terry): Use a more adaptive layout than forcing to 300.0
+        width: defaultSearchTextWidth,
```

Comment:
consider using `wideSearchTextWidth` since there is more screen real estate now

Reply:
Done.